src/ocm/components/clusterConfiguration/platformIntegration/PlatformIntegration.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { OcmSwitchField } from '../../ui/OcmFormFields';
+import { isClusterPlatformTypeVM, Cluster, PopoverIcon, PlatformType } from '../../../../common';
+import useClusterSupportedPlatforms, {
+  SupportedPlatformIntegrationType,
+} from '../../../hooks/useClusterSupportedPlatforms';
+import PlatformIntegrationVsphere from './PlatformIntegrationVsphere';
+
+type KeyType = Record<SupportedPlatformIntegrationType, string>;
+
+const undeterminedPlatformMessage =
+  'Enable platform integration to access features directly in OpenShift, like vSphere node auto-scaling and persistent storage or Nutanix node auto-scaling.';
+const vspherePlatformMessage =
+  "Enable vSphere platform integration to access features directly inside OpenShift, like node auto-scaling and persistent storage. You'll need to set vSphere configuration after cluster installation is complete.";
+
+const nutanixPlatformMessage =
+  'Enable Nutanix platform integration to access features like node auto-scaling directly inside OpenShift.';
+
+const platformIntegrationTooltip =
+  'Platform integration is only applicable when all discovered hosts originated from the same platform.';
+
+const messages: KeyType = {
+  'no-active-integrations': undeterminedPlatformMessage,
+  vsphere: vspherePlatformMessage,
+  nutanix: nutanixPlatformMessage,
+};
+
+const PlatformIntegrationLabel = ({
+  supportedPlatformIntegration,
+}: {
+  supportedPlatformIntegration: SupportedPlatformIntegrationType;
+}) => {
+  return (
+    <>
+      <span>Integrate with platform (vSphere/Nutanix)</span>{' '}
+      <PopoverIcon
+        bodyContent={messages[supportedPlatformIntegration]}
+        footerContent={supportedPlatformIntegration === 'vsphere' && <PlatformIntegrationVsphere />}
+        buttonOuiaId="platform-integration-enabled-popover"
```

Comment:
I know I did this change myself, but now I've realised that this is a "ouiaId" and not a "data-testid".
No it's best no to change it, as we'd need to change the ID somewhere in Pendo and I'm not aware of how to do it.